raptorWeb/authprofiles/views.py

```diff
@@ -0,0 +1,347 @@
+                                "mc_username": user_extra.minecraft_username,
+                                "favorite_pack": user_extra.favorite_modpack
+                            }
+                        }
+                    })
+                else:
+                    instance_dict.update({
+                        "displayed_profile": {
+                            "base": {
+                                "username": user_base.username,
+                                "date_joined": user_base.date_joined,
+                                "last_login": user_base.last_login,
+                                "is_staff": user_base.is_staff
+                            },
+                            "extra": {
+                                "picture": f'https://{settings.DOMAIN_NAME}/media/profile_pictures/{user_extra.profile_picture.name.split("/")[1]}',
+                                "mc_username": user_extra.minecraft_username,
+                                "favorite_pack": user_extra.favorite_modpack
+                            }
+                        }
+                    })
+            except IndexError:
+                instance_dict.update({
+                "displayed_profile": {
+                    "base": {
+                        "username": user_base.username,
+                        "date_joined": user_base.date_joined,
+                        "last_login": user_base.last_login,
+                        "is_staff": user_base.is_staff
+                    },
+                    "extra": {
+                        "mc_username": user_extra.minecraft_username,
+                        "favorite_pack": user_extra.favorite_modpack
+                    }
+                }
+            }) 
+        except User.DoesNotExist:
+            try:
+                discord_user = DiscordUserInfo.objects.get(username=profile_name)
+                instance_dict.update({
+                    "displayed_profile": {
+                        "base": {
+                            "username": discord_user.username,
+                            "date_joined": discord_user.date_joined,
+                            "last_login": discord_user.last_login,
+                        },
+                        "extra": {
+                            "picture": discord_user.profile_picture,
+                            "mc_username": discord_user.minecraft_username,
+                            "discord_username": discord_user.tag,
+                            "favorite_pack": discord_user.favorite_modpack
+                        }
+                    }
+                })
+            except DiscordUserInfo.DoesNotExist:
+                return HttpResponse("A User with the provided username was not found")
+
+        return render(request, self.template_name, context=instance_dict)
+
+class User_Profile_Edit(LoginRequiredMixin, TemplateView):
```

Comment:
The same issues with line 163 review for the `User_Profile` view apply to here at `User_Profile_Edit`. Too much code duplication, it seems a simpler approach may be available.